server/webhook_markdown.go

```diff
@@ -0,0 +1,189 @@
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
+// See License for license information.
+
+package main
+
+import (
+	"fmt"
+	"io"
+	"strings"
+
+	"github.com/andygrunwald/go-jira"
+
+	"github.com/mattermost/mattermost-server/model"
+)
+
+const (
+	mdRootStyle   = "## "
+	mdUpdateStyle = "###### "
+)
+
+func AsMarkdown(in io.Reader, n notifier, ji Instance) (func(post *model.Post), error) {
```

Comment:
This isn't used somewhere. Is this still needed?

Reply:
Yes, I'd like to keep it as an alternative renderer. This code will soon be refactored again, for the issue preview.